src/TextToSpeech.hs

```diff
@@ -51,12 +51,22 @@ espeak_contrib myDir lang tmp txt =
     l = case lang of
             Language s    -> s
 
+say :: Language -> FilePath -> String -> (String, [String])
+say lang tmp txt =
+   ("say", ["-o", tmp, "--data-format=LEF32@8000", "-v", l, txt])
+  where
+   l = case lang of
+           Language "en" -> "Alex"
+	   Language "de" -> "Anna"
+	   Language "fr" -> "Thomas"
+	   Language s    -> s
```

Comment:
This means that if any other language is used, it will passed to `say` as the `-v` parameter, probably not what you want.

You can replace `-> s` with `error "This language is not supported by the \"say\" text-to-speech engine"`.